config.py

```diff
@@ -21,4 +21,7 @@ def get_channel_reaper_settings():
         'whitelist_keywords': os.environ.get('WHITELIST_KEYWORDS', ''),
         'skip_subtypes': {'channel_leave', 'channel_join'},
         'skip_channel_str': os.environ.get('SLACK_SKIP_PURPOSE', '%noarchive'),
+        'max_channels_reviewed': int(os.environ.get('CHANNELS_REVIEWED_MAX', 0)),
```

Comment:
I personally get confused when I use env vars and variables that are the same thing, but have different names.
Would calling the env var `MAX_CHANNELS_REVIEWED` make sense?
(and similar for the below?)